packages/react-apps-toolkit/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "react-apps-toolkit",
+  "version": "0.0.1",
+  "description": "Toolkit for building a Contentful app in React",
+  "keywords": [],
+  "author": "Contentful GmbH",
+  "license": "MIT",
+  "engines": {
+    "node": ">=14",
+    "npm": ">=6"
+  },
```

Comment:
Remove. We don't build a node library but a frontend library. We therefore don't have any engine requirements